Makefile

```diff
@@ -46,17 +46,30 @@ lint: build-static-linter
 build-certs-gen:
 	cd $(CERTS_GEN_DIR) && go build -o ./$(CERTS_GEN_NAME) .
 
-certs-gen: build-certs-gen
+rsa-certs-gen: build-certs-gen
 	@echo "Generating private and public keys for asymmetric encryption..."
 	$(CERTS_GEN_DIR)/$(CERTS_GEN_NAME)
 
-openssl-certs-gen:
+openssl-rsa-certs-gen:
 	@echo "Generating private and public keys for asymmetric encryption with openssl..."
 	openssl genpkey -algorithm RSA -out $(CERTS_DIR)/$(PRIVATE_KEY_PKCS8) -pkeyopt rsa_keygen_bits:$(KEY_SIZE)
 	openssl rsa -in $(CERTS_DIR)/$(PRIVATE_KEY_PKCS8) -out $(CERTS_DIR)/$(PRIVATE_KEY) -traditional
 	rm $(CERTS_DIR)/$(PRIVATE_KEY_PKCS8)
 	openssl rsa -pubout -in $(CERTS_DIR)/$(PRIVATE_KEY) -out $(CERTS_DIR)/$(PUBLIC_KEY)
 
+openssl-tls-certs-gen:
+	openssl req -x509 -newkey rsa:2048 -nodes -days 365 -keyout $(CERTS_DIR)/ca-key.pem -out $(CERTS_DIR)/ca-cert.pem -subj "/C=RU/ST=Russia/L=Moscow/O=DEV/OU=DEV/CN=CA/emailAddress=metrics@yandex.ru"
+	openssl req -new -keyout $(CERTS_DIR)/server-key.pem -out $(CERTS_DIR)/server-req.pem -config server-cert.cnf
+	openssl x509 -req -in $(CERTS_DIR)/server-req.pem -CA $(CERTS_DIR)/ca-cert.pem -CAkey $(CERTS_DIR)/ca-key.pem -CAcreateserial -out $(CERTS_DIR)/server-cert.pem -days 365 -extfile server-cert.cnf -extensions req_ext
+	openssl req -newkey rsa:2048 -nodes -keyout $(CERTS_DIR)/client-key.pem -out $(CERTS_DIR)/client-req.pem -subj "/C=RU/ST=Russia/L=Moscow/O=DEV/OU=DEV/CN=CA/emailAddress=metrics@yandex.ru"
+	openssl x509 -req -in $(CERTS_DIR)/client-req.pem -CA $(CERTS_DIR)/ca-cert.pem -CAkey $(CERTS_DIR)/ca-key.pem -CAcreateserial -out $(CERTS_DIR)/client-cert.pem -days 60
+
+proto-gen:
+	protoc -I=proto --go_out=./proto/gen --go_opt=paths=source_relative \
```

Comment:
Please integrate `buf` https://buf.build/docs/ecosystem/cli-overview
Please add `buf lint` in CI https://github.com/bufbuild/buf-lint-action